components/merkledb/src/access/mod.rs

```diff
@@ -162,7 +162,7 @@ impl<T: RawAccess> Access for T {
 /// # Examples
 ///
 /// ```
-/// use exonum_merkledb::{access::{AccessExt, AccessRefExt, Prefixed}, Database, TemporaryDB};
+/// use exonum_merkledb::{access::{CopyAccessExt, AccessExt, Prefixed}, Database, TemporaryDB};
```

Comment:
Nit: Non-alphabetical ordering between `CopyAccessExt` and `AccessExt` (in some other places as well).